packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_flutter_web.dart

```diff
@@ -362,6 +373,7 @@ class GoogleMapsPlugin extends GoogleMapsFlutterPlatform {
     GoogleMapsInspectorPlatform.instance = GoogleMapsInspectorWeb(
       (int mapId) => _map(mapId).configuration,
       (int mapId) => _map(mapId).clusterManagersController,
+      (int mapId) => _map(mapId),
```

Comment:
Now that we're passing the whole map controller, the constructor for the `GoogleMapsInspectorWeb` could be simplified to just accept the controller, and have it internally extract `.configuration` and `.clusterManagersController`, as needed, I think.

Reply:
I think this is fine, that way we don't accumulate too much technical debt from the web implementation, or have too many `skip: kIsWeb` tests in the app-facing package! Thanks!